### ISSUE TITLE
HostedHomeserver

### ISSUE THREAD

**Issue by bmarty** [vector-im/riotX-android#66]
Manage quota (M_LIMIT_EXCEEDED error)

**Comment by mnaturel** [comment]
It seems this type of server error is handled now in the current version.  @bmarty Can you confirm we can close this issue?

**Comment by bmarty** [comment]
I do not think this has been implemented yet. FTR the Riot corresponding issue was https://github.com/vector-im/riot-android/issues/2440.

I do not know if we need to implement this on Element Android and its SDK nowadays.

CC @rxl881 WDYT?
